cocos/tween/actions/action.ts

```diff
@@ -215,6 +213,10 @@ export abstract class Action {
         return this._id;
     }
 
+    setPaused (paused: boolean): void {
+        this._paused = paused;
+    }
```

Comment:
Why not use setter?

Reply:
Just keep the same API style in `Action` class.

Reply:
Ok.